File: C_Programming/Day_4/program20.c
//Description : Write a program that accepts number from the user & count its 
//              even digits


#include<stdio.h>
#include<stdbool.h>

int CountEvenDigits(int iNo)
{
    int iDigit = 0;
    int iCount = 0;

    while(iNo != 0)
    {
        iDigit = iNo % 10;
        if(iDigit % 2 == 0)
        {
            iCount++;
        }
        
        iNo = iNo / 10;
    }

    return iCount;

}

int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter the number : ");
    scanf("%d",&iValue);

    iRet = CountEvenDigits(iValue);

    printf("Number of even digits are : %d\n",iRet);

    return 0;
}

File: Python_Programming/Day_4/program20.py
# Description : Write a program tha accepts number from the user & count its 
#               even digits

def CountEvenDigits(No):
    Digit = 0
    CountEven = 0

    while(No != 0):
        Digit = No % 10
        if(Digit % 2 == 0):
            CountEven += 1
        No = No // 10

    return CountEven

def main():

    Value = 0
    Ret = 0

    print("Enter the number : ")
    Value = int(input())

    Ret = CountEvenDigits(Value)

    print("Number of digits are : ",Ret)

if __name__ == "__main__":
    main()